examples/aws/api_gateway_with_domain_mtls/providers.tf

```diff
@@ -0,0 +1,36 @@
+terraform {
+  required_providers {
+    aws = {
+      source  = "hashicorp/aws"
+      version = "~> 5.11.0"
+    }
+  }
+
+  required_version = "1.5.5"
+}
+
+// default provider that is used when no other provider
+// is specified explicitly
+provider "aws" {
+  region = "eu-west-1"
+
+  default_tags {
+    tags = {
+      Project   = "they-terraform-examples"
+      CreatedBy = "terraform"
+    }
+  }
+}
+
+// provider where domain certificates are stored
+provider "aws" {
+  alias  = "acm_region"
```

Comment:
I think this is a leftover and should be removed, no?

Reply:
Oh yes! you are absolutely correct.
good catch.

Reply:
fixed